pal/samples/esp8266/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.ino

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <AzureIoTHub.h>
+
+// CAVEAT: This sample is to demonstrate azure IoT client concepts only and is not a guide design principles or style
+// Checking of return codes and error values shall be omitted for brevity.  Please practice sound engineering practices
+// when writing production code.
+
+// Note: PLEASE see https://github.com/Azure/azure-iot-arduino#simple-sample-instructions for detailed sample setup instructions.
+// Note2: To use this sample with the esp32, you MUST build the AzureIoTSocket_WiFi library by using the make_sdk.py,
+//        found in https://github.com/Azure/azure-iot-pal-arduino/tree/master/build_all. 
+//        Command line example: python3 make_sdk.py -o <your-lib-folder>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+// You must set the device id, device key, IoT Hub name and IotHub suffix in
+// iot_configs.h
+#include "iot_configs.h"
+#include "sample_init.h"
+#ifdef is_esp_board
+  #include "Esp.h"
+#endif
+
+
+static char ssid[] = IOT_CONFIG_WIFI_SSID;
+static char pass[] = IOT_CONFIG_WIFI_PASSWORD;
+
+#ifdef SAMPLE_MQTT
+    #include "AzureIoTProtocol_MQTT.h"
+    #include "iothubtransportmqtt.h"
+#endif // SAMPLE_MQTT
+#ifdef SAMPLE_HTTP
+    #include "AzureIoTProtocol_HTTP.h"
+    #include "iothubtransporthttp.h"
+#endif // SAMPLE_HTTP
+
+/* Define several constants/global variables */
+static const char* connectionString = DEVICE_CONNECTION_STRING;
+#define MESSAGE_COUNT 5 // determines the number of times the device tries to send a message to the IoT Hub in the cloud.
+static bool g_continueRunning = true; // defines whether or not the device maintains its IoT Hub connection after sending (think receiving messages from the cloud)
+static size_t g_message_count_send_confirmations = 0;
+
+IOTHUB_MESSAGE_HANDLE message_handle;
+size_t messages_sent = 0;
+const char* telemetry_msg = "test_message";
+
+// Select the Protocol to use with the connection
+#ifdef SAMPLE_MQTT
+    IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol = MQTT_Protocol;
+#endif // SAMPLE_MQTT
+#ifdef SAMPLE_HTTP
+   IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol = HTTP_Protocol;
+#endif // SAMPLE_HTTP
+
```

Comment:
Lets move this to setup function like we do in other samples>

[https://github.com/Azure/azure-iot-sdk-c/blob/2941f995704e4e1df8ec7565da234aa2011f93df/iothub_client/samples/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.c#L89](https://github.com/Azure/azure-iot-sdk-c/blob/2941f995704e4e1df8ec7565da234aa2011f93df/iothub_client/samples/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.c#L89)